.buildkite/scripts/cloud_tests.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+# What Terraform Module will run
+export MODULE_DIR="x-pack/metricbeat/module/aws"
+
+source .buildkite/scripts/install_tools.sh
+
+set -euo pipefail
+
+trap 'teardown; unset_secrets' EXIT
+
+# Set the MODULE env variable if possible
+defineModuleFromTheChangeSet "${MODULE_DIR}"
```

Comment:
IIUC, this particular function should use the beats project folder. 

`directory` in Jenkinsfile is set in https://github.com/elastic/beats/blob/3f462221002e595c5763bc626f1dc1a8e1b508e2/Jenkinsfile#L1133

and https://github.com/elastic/beats/blob/3f462221002e595c5763bc626f1dc1a8e1b508e2/Jenkinsfile#L1105 explains what the `project` means.

Then `directory` is used in https://github.com/elastic/beats/blob/3f462221002e595c5763bc626f1dc1a8e1b508e2/Jenkinsfile#L676

Reply:
I've changed the function and its call: https://github.com/elastic/beats/pull/38081/commits/141c990fb8b80def88fed8bfbbe242bb456d4f1a
https://github.com/elastic/beats/pull/38081/commits/3cf8aa0bc367dbb61cf4693227ab0325cf77babf